src/whisper-flat.cpp

```diff
@@ -0,0 +1,60 @@
+#include "whisper.h"
```

Comment:
Could the number of includes be reduces here to only `whisper-flat.h` and `ggml-backend.h` perhaps?

Reply:
Ahh, yeah - probably overkill there :)

I just copied what was in whisper.cpp and added to it (lazy me, sorry)